server/src/callbacks.mjs

```diff
@@ -47,10 +47,11 @@ Empirica.onGameStart(function ({ game }) {
   round.set("topic", game.batch.get("topics")[url]);
   const players = game.players
   const ids = []
```

Comment:
is the ids variable still being used? if not, lets pull it out.

Reply:
Yes, it gets used to set the gamestartids so that gameStart can be easily linked to the players in the game for data analysis